bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/MathUtils.java

```diff
@@ -28,6 +28,11 @@
 
     public static long divide(long numerator, long denominator) {
         try {
+            if (numerator == Math.pow(-2, 63) && denominator == -1) {
```

Comment:
Integer.MIN_VALUE

Reply:
Do you mean java int min value? that is -2^31.

Reply:
I think i should use Long.Min value. :)

Reply:
Yes the Long version :D

Reply:
Fixed. Thanks